handbrake.json

```diff
@@ -5,9 +5,9 @@
                 "image": "jlesage/handbrake",
                 "launch_order": 1,
                 "devices": {
-                    "VAAPI": {
-                        "description": "<u>Optional:</u> path to hardware transcoding device (/dev/dri/renderD128). Leave blank if not needed.",
-                        "label": "VAAPI device"
+                    "quicksync": {
```

Comment:
my only thing here is future support for AMD APU's. But I guess that can become a TODO:

Reply:
@mikey0000 , thanks a lot for your prompt feedback!

Is VAAPI actually supported already? I could only see mention of QuickSync in the image's doc so my understanding was that only QuickSync was supported so far.
If VAAPI is supported as well, please accept my apologies and I'll revert these changes and put the QuickSync label in _in addition to_ rather than _instead of_ VAAPI.

Reply:
Ah sorry I meant I'll need to look at adding support for AMD GPU/APU's as @phillxnet said can make the adjustment in a revised version.